src/Components/VideoPlayer/useInterval.tsx

```diff
@@ -0,0 +1,21 @@
+import { useRef, useEffect } from 'react'
+function useInterval(callback: () => void, delay: number | null) {
```

Comment:
Does this come from https://usehooks-typescript.com/react-hook/use-interval? If so, add that as a comment at the top of the file